src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+{% endAside %}
+
+## What is the Handwriting Recognition API? {: #what }
+
+> TODO: GIF of demo
+
+The Handwriting Recognition API is a new web platform API that allows you to convert handwritten
+drawings (ink) from your users into text. Some operating systems have long included such APIs,
+and with this new capability, your web apps can finally leverage the same powerful functionality.
+The conversion takes place directly on the user's device, works even in offline mode, all without
+adding any third-party libraries or services.
+
+This API implements a so-called "on-line" or real-time recognition. This means that the handwritten
+input is recognized while the user is drawing it, by capturing and analyzing the single strokes. In
+contrast to "off-line" procedures such as Optical Character Recognition (OCR), where only the end
+product is known, on-line algorithms can provide a higher level of accuracy.
+
+### Suggested use cases for the Handwriting Recognition API {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Note-taking applications where users want to capture handwritten notes and have them translated
+  into text.
+- Forms applications where users use pen or finger input due to time constraints.
+- Games that require filling in letters, such as crosswords or Hangman.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use Handwriting Recognition {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Handwriting Recognition API locally, without an origin trial token, enable
+the `#experimental-web-platform-features` flag in `chrome://flags`.
+
+{% Aside %} Note that the API is currently exclusive to Chrome&nbsp;OS devices. {% endAside %}
+
+> REVIEW: Add the version? Probably M90+
+
+> REVIEW: There is no Origin Trial yet.
+
+### Feature detection
+
+Browser support can be detected by checking for the existence of the `createHandwritingRecognizer()`
+method on the navigator object:
+
+```js
+function supportsHandwritingRecognition() {
+  return 'createHandwritingRecognizer' in navigator;
+}
+```
+
+### Core concepts
+
+> TODO: Image showing the core entities
+
+The Handwriting Recognition API converts handwritten input into text, regardless of the input
+method (mouse, touch, pen). The API has four main entities:
+
+1. A _point_ represents where (i.e., X and Y coordinates) the pointer was at a particular time.
```

Comment:
```suggestion
1. A _point_ represents where the pointer was at a particular time (i.e., X and Y coordinates).
```

Reply:
```suggestion
1. A _point_ represents where the pointer was at a particular time.
```

Time + X, Y coordinates doesn't make too much sense, maybe drop the entire coordinates thing?